contentcuration/kolibri_content/migrations/0002_auto_20180327_1414.py

```diff
@@ -9,26 +9,84 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('content', '0001_initial'),
+        ("content", "0001_initial"),
     ]
 
     operations = [
         migrations.AlterField(
-            model_name='file',
-            name='extension',
-            field=models.CharField(blank=True, choices=[('mp4', b'MP4 Video'), ('vtt', b'VTT Subtitle'), ('srt', b'SRT Subtitle'), ('mp3', b'MP3 Audio'), ('pdf', b'PDF Document'), ('jpg', b'JPG Image'), (
-                b'jpeg', b'JPEG Image'), ('png', b'PNG Image'), ('gif', b'GIF Image'), ('json', b'JSON'), ('svg', b'SVG Image'), ('perseus', b'Perseus Exercise'), ('zip', b'HTML5 Zip'), ('epub', b'ePub Document')], max_length=40),
+            model_name="file",
+            name="extension",
+            field=models.CharField(
+                blank=True,
+                choices=[
+                    ("mp4", "MP4 Video"),
+                    ("vtt", "VTT Subtitle"),
+                    ("srt", "SRT Subtitle"),
+                    ("mp3", "MP3 Audio"),
+                    ("pdf", "PDF Document"),
+                    ("jpg", "JPG Image"),
+                    (b"jpeg", "JPEG Image"),
```

Comment:
hmm `jpeg` is a bytestring, while the rest aren't. Is there a reason for that?

Reply:
Must have been missed by my grep - I imagine that maybe it was already `b"` instead of `b'` that I was searching for. Good catch!

Reply:
```suggestion
                    ("jpeg", "JPEG Image"),
```

Reply:
Added a commit to remove all the sneaky bs that made it through!